metatomic-torch/src/register.cpp

```diff
@@ -207,6 +207,7 @@ TORCH_LIBRARY(metatomic, m) {
 
     // standalone functions
     m.def("version() -> str", metatomic_torch::version);
+    m.def("pick_device(list supported_devices, str requested_device = None) -> torch.Device", metatomic_torch::pick_device);
```

Comment:
`str requested_device = None` feels wrong, should it be `Optional[str] requested_device = None` or `str requested_device? = None` (not sure about the syntax for optional here)